xds/internal/httpfilter/rbac/rbac.go

```diff
@@ -40,37 +40,40 @@ import (
 
 func init() {
 	if envconfig.XDSRBAC {
-		httpfilter.Register(builder{})
+		httpfilter.Register(Builder{})
```

Comment:
Why are we exporting this?  Can we not find it in the registry when it's needed by its name instead?